### DISCUSSION TITLE
页面报错但是指向mf-dep 开头的文件____vendor.3caaee3d.js:208792

### DISCUSSION THREAD

**Discussion by dddssw** [umijs/umi#12323] [Q&A]
页面报错是这样的
![image](https://github.com/umijs/umi/assets/58782768/611dc4b3-f871-416c-9834-4bef99f78ea5)
控制台报错
![image](https://github.com/umijs/umi/assets/58782768/710f3a72-f427-47b5-acc6-15ddd2b9fe3c)
点进去
![image](https://github.com/umijs/umi/assets/58782768/eeef4663-45c9-4d75-a8ce-2bdcf1150ae2)
我不知道问题出在哪里 另外mf-dep开头的文件是什么

**Comment by fz6m** [comment]
1. 你根据报错栈上最顶上的几条就可以定位到是哪里出问题了，比如这里说 `items.map` 是空的，所以才会 undefined 执行不了，位置在 `InternalUploadList` 这个组件，找到对应的位置排查下为什么会为空就行。

2. mf 开头的文件是开启了 mfsu （umi 4 默认情况会开启）后，为了加速项目启动而预打包的一些 js 的文件名（仅限开发时）。

**Reply by dddssw** [reply]
找了很久才知道， 这不是我的业务代码，而是我使用antd组件库内部的报错，

**Reply by fz6m** [reply]
你的这个应用是 umi 3 吗，umi 3 不维护了，建议升级到 umi 4 。

**Reply by dddssw** [reply]
我这个是umi4

**Reply by dddssw** [reply]
我找到了antd InternalUploadList这个组件的源码，这个错误的产生是我使用到了form，form里有一个upload组件，form.setFieldsValue时候给这个upload传了一个空字符串，但是应该是一个数组，产生了这个报错🤪
